### ISSUE TITLE
How to use the returned refresh token next time authentcate?

### ISSUE THREAD

**Issue by ArnabChatterjee20k** [sns-sdks/python-youtube#150]
I am generating refresh_token but no method or attribute is present in Api class to use it to authenticate next time using the refresh_token. How to use refresh_token for authentication?

**Comment by MerleLiuKun** [comment]
If you have `refresh_token`, you can use method `refresh_access_token` to get new access token.

**Comment by ArnabChatterjee20k** [comment]
> 

But how to use it to authenticate. Means in Api class we can provide only access_token attribute no attribute for providing the refresh_token

**Comment by MerleLiuKun** [comment]
I don't know your mean?

**Comment by maxoja** [comment]
The confusing point is that when I call `refresh_access_token()` it only returns with new access_token without new refresh_token, but I guess it's because we can keep reusing the old refresh_token received from he initial token generation repeatedly

**Comment by MerleLiuKun** [comment]
Yes, You are right,  The `refresh_token` can be alaways use before it been invalid.
